index.md

```diff
@@ -519,8 +519,9 @@ Signed,
 - Naglis Lesčinskas
 - Nathan Dyer (former FSF associate member)
 - Nathan Handler (Ubuntu/Debian Developer, freenode Staff, SeaGL Program Committee Co-Chair)
-- Nathaniel J. Smith
 - Nathan Ritchie
+- Nathan Shelly
+- Nathaniel J. Smith
```

Comment:
running `sort -u` on macOS reshuffled these. Please let me know if you'd prefer I remove these changes and just add my name!